app/config.matka.js

```diff
@@ -0,0 +1,87 @@
+const CONFIG = process.env.CONFIG || 'matka';
+const APP_DESCRIPTION =
+  'Liikenneviraston Matka.fi uudistuu. Apuasi kaivataan kehitystyössä. ' +
+  'Tule palvelun testaajaksi tai tee siitä saman tien parempi.';
+
+export default {
+  CONFIG,
+
```

Comment:
Should probably include 

`
  availableLanguages: ['fi', 'sv', 'en'],
  defaultLanguage: 'fi',
`